tensorflow/stream_executor/cuda/cuda_blas.cc

```diff
@@ -299,6 +305,17 @@ static string ToString(cublasStatus_t status) {
   }
 }
 
+// Decide whether to enable TENSOR_OP_MATH
+static bool TensorOpMathEnabled() {
+  static bool is_enabled = [] {
+    bool ret;
+    TF_CHECK_OK(tensorflow::ReadBoolFromEnvVar("TF_DISABLE_TENSOR_OP_MATH",
```

Comment:
I noticed that in cuda_dnn.cc, there is also an env-var with exactly the same name: TF_DISABLE_TENSOR_OP_MATH. Ideally, we would want each env-var to only control the behavior of one op, this will make debugging easier. Could you consider using two different names such as TF_DISABLE_CUBLAS_TENSOR_OP_MATH and TF_DISABLE_CUDNN_TENSOR_OP_MATH?

Reply:
TF_DISABLE_TENSOR_OP_MATH continues to disable tensor ops in both cublas and cudnn, but I added TF_DISABLE_CUBLAS_TENSOR_OP_MATH and TF_DISABLE_CUDNN_TENSOR_OP_MATH which will override the above and apply specifically to CUBLAS or CUDNN.